src/FakeItEasy/ExceptionMessages.tt

```diff
@@ -0,0 +1,122 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>

```

Comment:
I was surprised to see two templates. I'd've guessed that the one would do the job for all the resxes. Then again, I've never _used_ T4, only read about it.